octobot/config/profile_schema.json

```diff
@@ -138,6 +138,10 @@
               "type": "number",
               "minimum": 0,
               "maximum": 1
+            },
+            "current-bot-recording-id": {
+              "type": "integer",
+              "minimum": 1
```

Comment:
what is a bot recording id and what is it used for ?

Reply:
its used to switch the live run database (for plots and so on).
Its similar to the optimizer campaign id, but only for live trading

Reply:
Therefore I guess it's local to a profile and used for display only, right ? In this case I think it would be better to put it in the strategy designer configuration since it's only used by this (unless I'm missing something there).

Reply:
its also used for the portfolio value history json file that gets written every day, even without the strategy designer I think